templates/EXPERIMENTAL_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: default
+title: Experimental feature name
+nav_order: 
+parent: 
+---
+
+# Experimental feature name
+
+This is an experimental feature and is not recommended for use in a production environment. For updates on the progress of the feature or if you want to leave feedback, see the associated [GitHub issue](https://example.issue.link).    
```

Comment:
Should we separate these examples into two headers or introduce them? For example,

"The OpenSearch team announces and solicits feedback for experimental features using the [OpenSearch forum](https://forum.opensearch.org/). However, some experimental features are discussed inside a GitHub issue. Use one of the following templates depending on how OpenSearch developments wants to seek feedback from the community."

Reply:
I think it's pretty self explanatory so I vote for waiting if we get any comments on that. Otherwise, it's more text to delete.